bids/layout/layout.py

```diff
@@ -320,9 +320,9 @@ def get_metadata(self, path, include_entities=False, scope='all'):
         file = self.dataset.get_file(path)
         md = file.get_metadata()
```

Comment:
Artifact.get_metadata(include_entities=True) is now also possible, so no need to handle this manually here